### PR TITLE
Prompt user to remove frontail on openHAB 4.3+

### PR DESCRIPTION
**Pull request by ecdye**
I have opted not to add an unattended install option as there was not an option to disable it in the first place. I believe that it is appropriate that the user need to manually set it up using the menu if we are really not offering support for this anymore (beyond the obvious availability of the code as it was).

@mstormi let me know what you think.

### CODE REVIEW

**Comment by mstormi** [comment]
> have opted not to add an unattended install option

In general, all additions should be supporting both modes
I was thinking we would still need those parts whenever we come up with whatever replacement for frontail, but it's fine, too, to re-add the unattended options again then

**Review by mstormi — APPROVED** [review]
I didn't check the code, assuming you only pulled back in what was there before.

Maybe more prominently point at the new log viewer in OH?
See my annotations.
I'm sure most OH users will have missed that.

Other than that, LGTM

File: functions/nodejs-apps.bash [hunk 1]
```diff
@@ -54,32 +54,144 @@ nodejs_setup() {
...
 ##
 frontail_remove() {
   local frontailBase
   local frontailDir="/opt/frontail"
+  local removeText="Frontail is a log viewer that is not maintained and has security issues. As of openHAB 4.3 there is a built in log viewer which replaces it.\\n\\nWould you like to remove it from your system? If not, be aware that it is not recommended to use it and is no longer a supported feature of openHABian."
 
   frontailBase="$(npm list -g | head -n 1)/node_modules/frontail"
 
-  if ! [[ $(openhab-cli info | grep "Version" | xargs | cut -d ' ' -f 2) =~ 4.[3-9]* ]]; then return 0; fi
+  if ! dpkg --compare-versions "$(sed -n 's/openhab-distro\s*: //p' /var/lib/openhab/etc/version.properties)" gt "4.3.0"; then return 0; fi
```
[the remark this answers is not in the document]
Comment (ecdye) [reply functions/nodejs-apps.bash:83]: I tested it with some arbitrary values on my machine and it seems to.

File: NEWS.md [hunk 2]
```diff
@@ -2,12 +2,15 @@ Hit tab to unselect buttons and scroll through the text using UP/DOWN or
...
-present and is no longer maintained. We apologize for any inconvenience
-this may cause.
+## Frontail removed ## December 18, 2024
+We suggest removal of the frontail log-viewer package on all systems with
+openHAB 4.3+. There is still an option to keep it or install it however it
+is no longer supported and is provided as is. The reasoning for removal is
+that frontail has serious security vulnerabilities present and is no longer
+maintained.
+
+You can look forward to the builtin logviewer becoming even more capable
```
[the remark this answers is not in the document]
Comment (ecdye) [reply NEWS.md:12]: Sounds good, I'll commit these changes and forward to openHAB this evening so that users will have the option to install it again.

**Comment by ecdye** [comment]
> In general, all additions should be supporting both modes, with hindsight also to eventually replace the CLI by some Web interface some day. I was thinking we would still need those parts whenever we come up with whatever replacement for frontail, but it's fine, too, to re-add the unattended options again then.

I tend to agree, but I don't really want to add the extra effort in this case as it was formerly a part of the base install, and we are no longer supporting it. I'm inclined to treat it more like one of those optional components that needs user interaction before install (both for security, and because I don't want to do a bunch of support for it)

**Pull request merged**